bpy_speckle/utils/version_manager.py

```
@@ -23,25 +24,25 @@ def get_versions_for_model(account_id: str, project_id: str, model_id: str, sear
         if not account_id or not project_id or not model_id:
             print(f"Error: Invalid inputs - account_id: {account_id}, project_id: {project_id}, model_id: {model_id}")
             return []
-            
+
         # Get the account info
-        account = next((acc for acc in get_local_accounts() if acc.id == account_id), None)
+        account: Optional[Account] = next((acc for acc in get_local_accounts() if acc.id == account_id), None)
         if not account:
             print(f"Error: Could not find account with ID: {account_id}")
             return []
-            
+
         # Initialize the client
-        client = SpeckleClient(host=account.serverInfo.url)
+        client: SpeckleClient = SpeckleClient(host=account.serverInfo.url)
         # Authenticate
         client.authenticate_with_account(account)
-        
-        filter = ModelVersionsFilter(search=search, priorityIds=[])
+
+        filter: ModelVersionsFilter = ModelVersionsFilter(search=search, priorityIds=[])
 
         # Get versions
-        versions = client.version.get_versions(project_id=project_id, model_id=model_id, limit=10, filter=filter).items
-            
+        versions: List[Version] = client.version.get_versions(project_id=project_id, model_id=model_id, limit=10, filter=filter).items
+
         return [(version.id, version.message or "No message", format_relative_time(version.createdAt)) for version in versions]
-        
+
     except Exception as e:
         print(f"Error fetching versions: {str(e)}")
-        return [] 
\ No newline at end of file
+        return []
```

This is proably ok for now, 
but returning a valid value on error is usually a bad pattern for error handling. I would prefer to throw an exception here, document that exception in the function's doc string, and let the caller decide how to handle it.

---

noted.